src/dotnet-format.csproj

```diff
@@ -29,6 +29,7 @@
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="$(MicrosoftExtensionsDependencyInjectionVersion)" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsLoggingVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.CodingConventions" Version="$(MicrosoftVisualStudioCodingConventionsVersion)" Publish="true" />
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVerison)" />
```

Comment:
I could have used [System.Text.Json](https://docs.microsoft.com/en-us/dotnet/api/system.text.json?view=netcore-3.0) for this, but that requires .Net Core 3.0.

I could upgrade the .Net Core version if you'd like, but didn't want to do that unilaterally.

Reply:
I would appreciate it if you upgraded the project to 3.1 and useing System.Text.Json

Reply:
this is actually more involved that I though. I've done it in this PR: https://github.com/dotnet/format/pull/496

Reply:
actually scratch that. You should be able to reference the nuget package for `System.Text.Json` and use it already